src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Parser/KnownTypeSymbols.cs

```diff
@@ -132,6 +133,9 @@ public KnownTypeSymbols(CSharpCompilation compilation)
             IntPtr = Compilation.GetSpecialType(SpecialType.System_IntPtr);
             UIntPtr = Compilation.GetSpecialType(SpecialType.System_UIntPtr);
             Delegate = Compilation.GetSpecialType(SpecialType.System_Delegate);
+
+            // Only generate nullable attributes if available
+            NotNullIfNotNullAttribute = compilation.GetBestTypeByMetadataName(typeof(System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute));
```

Comment:
>typeof(System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute) [](http://example.com/codeflow?start=78&length=65)

please convert this to a string `System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute` instead of using typeof.